docs/dev/cli/sauce-connect-5/sc_run.md

```diff
@@ -1,493 +1,385 @@
 ---
 id: run
 title: sc run
-sidebar_label: sc run
 ---
 
-import useBaseUrl from '@docusaurus/useBaseUrl';
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
+# Sc Run
 
-The `sc run` command is a main Sauce Connect Proxy 5 command that allows provisioning a Sauce Connect Proxy server and establishing a secure connection between the Sauce Connect Proxy client and the server.
+Usage: `sc run --username <username> --access-key <UUID> --region <data center> --tunnel-name <value> [flags]`
 
-## Usage
+Run Sauce Connect Proxy
 
-```bash
-SAUCE_USERNAME=<username> SAUCE_ACCESS_KEY=<access key> sc run --region <region> --tunnel-name my-tunnel [OPTIONS]
-```
-
-## Options
-
-### Main
-
----
+**Note:** You can also specify the options as YAML, JSON or TOML file using `--config-file` flag.
+You can generate a config file by running `sc run config-file` command.
 
-#### `--username`
 
-<p><small>| REQUIRED | STRING | </small></p>
+## Required
 
-**Description**: Sets your Sauce Labs username.<br/>
-**Default**: n/a<br/>
-**Environment variable**: `SAUCE_USERNAME`<br/>
-**Shorthand**: `-u`
+### `-k, --access-key` {#access-key}
 
-:::note
-For additional security, we recommend setting this as an [environment variable](/secure-connections/sauce-connect-5/operation/configuration/#environment-variables).
-:::
+* Environment variable: `SAUCE_ACCESS_KEY`
+* Value Format: `<UUID>`
 
----
+Sauce Labs Access Key, you can get it from the User Settings page https://app.saucelabs.com/user-settings.
+For additional security, we recommend setting this as an environment variable.
 
-#### `--access-key`
+### `-r, --region` {#region}
 
-<p><small>| REQUIRED | STRING | </small></p>
+* Environment variable: `SAUCE_REGION`
+* Value Format: `<data center>`
 
-**Description**: Sets your Sauce Labs access key. This will be the same as your [Access Key](https://app.saucelabs.com/user-settings).<br/>
-**Default**: n/a<br/>
-**Environment variable**: `SAUCE_ACCESS_KEY`<br/>
-**Shorthand**: `-k`
+Sauce Labs region name, ex.
+us-west or eu-central.
+More details at https://docs.saucelabs.com/basics/data-center-endpoints.
 
-:::note
-For additional security, we recommend setting this as an [environment variable](/secure-connections/sauce-connect/setup-configuration/environment-variables/).
-:::
+### `-i, --tunnel-name` {#tunnel-name}
 
----
+* Environment variable: `SAUCE_TUNNEL_NAME`
+* Value Format: `<name>`
 
-#### `--region`
+Name of the tunnel or tunnel pool.
```

Comment:
@Choraden 

These links should be relative - we've got some tooling for detecting broken links etc, and having an absolute link to docs.saucelabs.com will cause problems.

Reply:
Actually, I can replace it, and trigger from a branch.